virttest/utils_v2v.py

```diff
@@ -742,31 +742,45 @@ def import_vm_to_ovirt(params, address_cache, timeout=600):
     return True
 
 
-def check_log(log, content_list, expect=True):
+def check_log(params, log):
     """
-    Check if Error Message exists in v2v log
+    Check if error/warning meets expectation in v2v log
 
     :param log: The log string to be checked
-    :param content_list: Content to be searched from log
-    :param expect: True if expect to find message from log, otherwise False
     :return: True if search result meets expectation, otherwise False
     """
-    found = False
-    for content in content_list:
-        line = '\s*'.join(content.split())
-        logging.info('Searching for: %s' % content)
-        pattern = re.compile(line)
-        search = re.search(pattern, log)
-        if search:
-            # Found log not expect
-            if not expect:
-                logging.error('Found error log of virt-v2v: %s' %
-                              search.group(0))
-                return False
-            found = True
-            logging.info('Found log: \n%s' % search.group(0))
-    # Expect to find log but failed
-    if not found and expect:
-        logging.error('Log not found: %s' % '\n'.join(content_list))
-        return False
-    return True
+
+    def _check_log(content_list, expect=True):
+        for content in content_list:
```

Comment:
Please don't split content and match it line by line, please use re.M to match multi-lines.